modules/recon/profiles-contacts/bing_linkedin_contacts.py

```diff
@@ -35,7 +36,9 @@ def get_contact_info(self, url):
             # Split the title on the pipe to get rid of "linkedIn" portion at the end
             name_and_title = link_title.split("|")[0]
             # Split whats left on the Dashes, which is usually name - title - company
-            name_title_company_list = name_and_title.split("-")
+            # some european LinkedIn sites use em-dash
+            delimeter_expression = '- | ' + b'\xe2\x80\x93'.decode('utf-8')
```

Comment:
Can we have the em-dash code as a constant for readability?

Reply:
Yes, that makes sense.